lib/wallet/wallet_lightning.dart

```diff
@@ -91,21 +113,24 @@ class WalletLightning extends StatelessWidget {
         onRefresh: _pullRefresh,
         child: ListView(padding: const EdgeInsets.only(left: 25, right: 25), children: widgets),
       )),
-      floatingActionButton: SpeedDial(
-        icon: Icons.import_export,
-        iconTheme: const IconThemeData(size: 35),
-        activeIcon: Icons.close,
-        activeBackgroundColor: Colors.grey,
-        activeForegroundColor: Colors.white,
-        buttonSize: const Size(56.0, 56.0),
-        visible: !hasOpenCfds,
-        closeManually: false,
-        curve: Curves.bounceIn,
-        overlayColor: Colors.black,
-        overlayOpacity: 0.5,
-        elevation: 8.0,
-        shape: const CircleBorder(),
-        children: dials,
+      floatingActionButton: Visibility(
+        visible: showActionButton,
```

Comment:
nit: we don't need that variable, you could use something like `channel.isUnavailable()`